dulwich/tests/test_porcelain.py

```diff
@@ -1842,13 +1842,18 @@ def test_status_all(self):
         mod_path = os.path.join(self.repo.path, "bar")
         add_path = os.path.join(self.repo.path, "baz")
         us_path = os.path.join(self.repo.path, "blye")
-        ut_path = os.path.join(self.repo.path, "blyat")
+        ut_path = os.path.join(self.repo.path, "untracked_file")
+        os.mkdir(os.path.join(self.repo.path, "untracked_dir"))
+        ut_subfile_path = os.path.join(self.repo.path, "untracked_dir/file")
```

Comment:
```suggestion
        ut_subfile_path = os.path.join(self.repo.path, "untracked_dir", "file")
```
hence the failing windows test

Reply:
Fixed everywhere